staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -225,6 +235,9 @@ type CPUStats struct {
 	// Cumulative CPU usage (sum of all cores) since object creation.
 	// +optional
 	UsageCoreNanoSeconds *uint64 `json:"usageCoreNanoSeconds,omitempty"`
+	// CPU PSI stats.
+	// +optional
+	PSI *cadvisorapi.PSIStats `json:"psi,omitempty"`
```

Comment:
I'm surprised to see us embedding another project's API in our API surface… does this mean anything cadvisor adds we have to support and surface?

Reply:
Yes, good point. We need to decouple from cadvisor and maintain our own type here.

Reply:
Done. Thanks for catching this!